src/elm/Main.elm

```diff
@@ -226,6 +229,20 @@ update msg model =
                             ]
                         )
 
+                GA.OpenEditTravelCard ->
+                    let
+                        accountModel =
+                            model.account
+
+                        newModel =
+                            { accountModel | editSection = Just AccountPage.TravelCardSection }
+                    in
+                        ( { model | account = newModel }
+                        , Cmd.batch
```

Comment:
No need to use `Cmd.batch` here.

Reply:
Hadde ikke oppdaga TaskUtils enda 😬  Fikser

Reply:
Ah. Sorry. Route.newUrl kan brukes direkte ja